boltdesignsystem/bolt
# Fix the Doc Site's broken off-canvas nav panel

## Jira
N/A

## Summary
Fixes the docs site's off-canvas navigation to not be 100% broken on smaller screens + adds E2E integration test to make sure this type of thing doesn't happen again.

## Details
Off-canvas nav broke when the new SVG icon system started to include an element with the same `#menu` ID 🤦‍♂ 

## How to test
- Confirm doc site's off-canvas nav panel now works (opens and closes) on smaller screen sizes
- Confirm that the new E2E integration test added here passes
